packages/strapi-plugin-upload/services/image-manipulation.js

```diff
@@ -87,12 +87,17 @@ const optimize = async buffer => {
     .catch(() => ({ buffer }));
 };
 
-const BREAKPOINTS = {
+const DEFAULT_BREAKPOINTS = {
   large: 1000,
   medium: 750,
   small: 500,
 };
 
+const BREAKPOINTS = strapi.config.get(
```

Comment:
can you wrap this into a getBreakpoints function & call it where you need the breakpoints. I would like to avoid any calls to strapi.config at the root of the modules as it would break as soon as we change our load algorithms :)

Reply:
Okay, done!